project_one/style.css

```diff
@@ -0,0 +1,89 @@
+#well{
+display: flex; 
+font-size: 60px;
+justify-content:space-around;
+color :red;
+
+}
+#grading-element-one{
+    background-color:#7c6f6f;
+    height: 20px;
+    margin-top: -1px
+}
+#grading-element-two-consequence{
+font-size: 20px;
+color: white;
+background-color:#745151;
+height: 700px;
+margin-top: -1px
+}
+
+#grading-element-four{
+display: flex;
+flex-wrap: wrap;
+align-content: space-around;
+flex-direction: column;
+height: 900px;
+}
+
+#soumettre{
+    background-color: greenyellow;
+    height: 400px;
+}
+
+
+#toucolor{
+    background-color:#745151;
+}
+#grading-element-three{
+    background-color: greenyellow;
+
+    
+
+
+}
+#soumettre{
+    background-color: greenyellow;
+    height: 50px;
+    width: 20%;
+
+
+}
+.bienvenue{
+    display: flex;
+    width: 100%;
+    height: 70%;
+}
+
+.photo{
+    display: flex;
+       width: 100%;
+       height: 70%;
+   }
+   
+   .photo{
+    display: flex;
+    width: 25%;
+    height: 25%;
+   
+   }
+   .Photos{
+       display: flex;
+       flex-wrap: wrap;
+       flex-direction: row;
+   }
+  
+
+   #grading-element-five:hover{
+    background-color: #7c6f6f;
+
+
+    color: #c41313;
+
+
+   }
+   .grading-element-three-consequence{
```

Comment:
this is a class selector, but `grading-element-three-consequence` is an `id`, so this css rule does nothing.